src/exampleTypes/ResultTable.js

```diff
@@ -105,6 +105,7 @@ let Header = withStateLens({ popover: false, adding: false, filtering: false })(
       criteria && _.find({ field }, tree.getNode(criteria).children)
     let filter = () => {
       if (!filterNode) addFilter(field)
+      filterNode.paused = false
```

Comment:
This is not safe to do because there could be pending updates. In general, we should never be modifying node properties outside of actions that dispatch the appropriate events. The right approach here is to do a mutate like

```js
tree.mutate(node.path, { paused: false })
```

You can see an example of us doing that in this repo here: https://github.com/smartprocure/contexture-react/blob/master/src/FilterList.js#L19